### PR TITLE
[Plugin] Add report for bazel build

### PR DESCRIPTION
**Pull request by gichan-jang**
Add a report for bazel build.
Currently, the log is included in `pr-postbuild-group.txt`

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

File: ci/taos/plugins-staging/pr-postbuild-bazel-ubuntu.sh [hunk 1]
```diff
@@ -116,10 +116,11 @@ function pr-postbuild-bazel-ubuntu-run-queue() {
     fi
 
     # Build a source code of the nnstreamer repository
-    echo -e "Starting a buile task with bazel command ..."
+    echo -e "Starting a build task with bazel command ..."
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-postbuild-bazel-ubuntu.sh:119]: Nice catch.

File: ci/taos/plugins-staging/pr-postbuild-bazel-ubuntu.sh [hunk 2]
```diff
@@ -116,10 +116,11 @@ function pr-postbuild-bazel-ubuntu-run-queue() {
...
     pwd
+    build_log=../report/build_log_${input_pr}_ubuntu_bazel_output.txt
     echo -e "statement (build): $START_BUILDTEST"
-    eval $START_BUILDTEST
+    eval $START_BUILDTEST >> $build_log
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-postbuild-bazel-ubuntu.sh:123]: Looks good.

**Review by leemgs — APPROVED** [review]
Signed-off-by: Geunsik Lim

**Pull request merged**